packages/browser-repl/package.json

```diff
@@ -38,69 +36,57 @@
     ]
   },
   "dependencies": {
+    "@mongodb-js/compass-components": "^1.3.0",
+    "@mongodb-js/compass-editor": "^0.2.0",
```

Comment:
Shouldn’t these be peerDependencies, maybe even with a `*` version, so that installing browser-repl in Compass doesn’t potentially result in these packages actually getting installed (as opposed to using the ones from the Compass monorepo)?

Reply:
Moving them to peer dependencies makes a lot of sense to me, yeah, will do! I am a bit hesitant to make them `*`, this always bothers me a bit too much as we can't really guarantee it (I know that we do breaking changes with minor patches quite often anyway sometimes, but still 😅), would it be okay with you if I leave them with `^` version or you have a strong preference for `*`?

Reply:
Ah, actually while these packages are still on `0.x` version, caret wouldn't allow them to resolve to the next minor patch because it's this pre-stable release case, right? So I guess `*` is the safest option here

Reply:
Yeah, that is (or at least used to ve the behavior here).

Ultimately there's a tradeoff here – if we go with *, we might not notice breaking changes, but otherwise we may need to do mongosh version bumps just to accommodate the compass bumps. I think * would be fine, but if you'd rather be more cautious, thats also very understandable.